packages/synapse-react-client/src/utils/hooks/useOneSageURL.ts

```diff
@@ -29,10 +31,9 @@ export function getOneSageBaseUrl(
     if (currentBackendEndpointWithoutSlash === SYNAPSE_BACKEND_STAGING_URL) {
       return ONE_SAGE_STAGING_URL
     }
-  }
-
-  if (currentHostname.endsWith('.sagebase.org')) {
-    // TODO: Return the dev accounts site created in IT-3849
+    if (currentBackendEndpointWithoutSlash === SYNAPSE_BACKEND_DEV_URL) {
+      return ONE_SAGE_DEV_URL
+    }
```

Comment:
Correctly route to `dev.accounts.synapse.org` when we try to send you to the accounts site